doc/classes/Dictionary.xml

```diff
@@ -202,6 +222,42 @@
 				Gets a value and ensures the key is set. If the [param key] exists in the dictionary, this behaves like [method get]. Otherwise, the [param default] value is inserted into the dictionary and returned.
 			</description>
 		</method>
+		<method name="get_typed_key_builtin" qualifiers="const">
+			<return type="int" />
+			<description>
+				Returns the [enum Variant.Type] constant for a typed dictionary key. If the [Dictionary] key is not typed, returns [constant TYPE_NIL].
+			</description>
+		</method>
+		<method name="get_typed_key_class_name" qualifiers="const">
+			<return type="StringName" />
+			<description>
+				Returns a class name of a typed [Dictionary] key of type [constant TYPE_OBJECT].
```

Comment:
```suggestion
				Returns the [b]built-in[/b] class name of the typed dictionary's keys, if the built-in [Variant] type is [constant TYPE_OBJECT]. Otherwise, returns an empty [StringName]. See also [method is_typed_key] and [method Object.get_class].
```